### PR TITLE
Fix my deadname

### PR DESCRIPTION
**Pull request by Gabriella439**

### CODE REVIEW

File: nix/shared.nix [hunk 1]
```diff
@@ -236,7 +236,7 @@ let
                   [ (pkgsNew.haskell.lib.packagesFromDirectory { directory = ./packages; })
                     extension
                     doCheckExtension
-                    doBenchmarkExtension
+                    # doBenchmarkExtension
```
Comment (SiriusStarr) [inline nix/shared.nix:239]: Should this be a separate PR?  Wasn't sure if you'd meant to include this change.

Comment (Gabriella439) [reply nix/shared.nix:239]: Yeah, that's a mistaken change that I accidentally committed.  I'll remove that
